src/stdlib/src/ascii_string.cc

```diff
@@ -157,8 +192,8 @@ namespace perlang
         size_t length = str.length() + rhs.length();
         char *bytes = new char[length + 1];
 
-        memcpy((void*)bytes, str.c_str(), str.length());
-        memcpy((void*)(bytes + str.length()), rhs.bytes(), rhs.length());
+        memcpy(bytes, str.c_str(), str.length());
+        memcpy((bytes + str.length()), rhs.bytes(), rhs.length());
```

Comment:
(Removed these `void*` casts both here and in `utf8_string.cc`. `char*` can be implicitly cast to `void*`, so let's perhaps use that fact to make the code a little bit more readable.